src/widgets.js

```diff
@@ -387,6 +387,37 @@ define([
         return widget;
     };
 
+    var button = function (mug, options) {
+        var widget = normal(mug, options),
+            disabled = options.disabled || false;
+        widget.input = $("<button />")
+            .addClass('btn btn-default')
+            .attr('type', 'button')
+            .attr("name", widget.id)
+            .attr("id", widget.id)
+            .prop('disabled', disabled)
+            .html(options.buttonContent);
+
+        widget.setValue = function (value) { };
+
+        widget.getValue = function() { return null; };
+
+        return widget;
+    };
+
+    var findUsagesButton = function (mug, options) {
```

Comment:
Surprised to see the button's action inside of the widget, I'd expected something like a generic button widget that accepts both a `buttonAction` function and `buttonContent`. On the other hand, there's not much point making this widget more generic until we have more than one use case for it.